ros_ethercat_model/include/ros_ethercat_model/robot_state.hpp

```diff
@@ -46,6 +46,7 @@
 #include <boost/ptr_container/ptr_vector.hpp>
 #include <hardware_interface/hardware_interface.h>
 #include "ros_ethercat_model/joint.hpp"
+#include "ros_ethercat_model/imu_state.hpp"
```

Comment:
This file is missing. Please add it.
I've created a task to add CI to this repository. This would have been detected.

Reply:
oops, good catch